alchemiscale/compute/client.py

```diff
@@ -31,6 +31,13 @@ class AlchemiscaleComputeClient(AlchemiscaleBaseClient):
 
     _exception = AlchemiscaleComputeClientError
 
+    def list_scopes(self) -> List[Scope]:
+        scopes = self._get_resource(
+            f"/identities/{self.identifier}/scopes",
+            return_gufe=False,
+        )
+        return [Scope.from_str(s) for s in scopes]
```

Comment:
I think it's more consistent for us to return `Scope` objects, since these can be directly used where a `Scope` is needed elsewhere, and it doesn't impact readability of printed results in my view.